anonklub/anonklub
# :bug: fix #123, download `@anonklub/prove` dep when building docker image

`@anonklub/prove` is now a published [npm package](https://www.npmjs.com/package/@anonklub/proof).  
So when building the prove-api docker image, we can download the dependency from the npm registry instead of using the workspace package. No need to copy `@anonklub/proof` into the docker image.  

Also updating the `apis/prove` README.

## Test plan:
From root folder:
1. ,Build the prove api image: `pnpm build.docker.prove`. There should be no failures.
2. Start redis: `pnpm --filter @anonklub/prove-api start.redis`
3. Start container: `docker run -p 3000:3000 prove-api`. There should be no errors.
4. Check server is running at `http://localhost:3000`
